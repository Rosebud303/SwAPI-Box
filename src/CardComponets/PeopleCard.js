import React from 'react';
import './PeopleCard.scss'

export default function PeopleCard({person}) {
  return (
    <div className='peopleCard'>
      <h3 className='peopleName'>{person.Name}</h3>
      <p className='peopleDescription'>Homeworld: {person.Homeworld}</p>
      <p className='peopleDescription'>Species: {person.Species}</p>
      <p className='peopleDescription'>Language: {person.Language}</p>
      <p className='peopleDescription'>Population: {person.Population}</p>
      <button>Add Favorite</button>
    </div>
  )
}
