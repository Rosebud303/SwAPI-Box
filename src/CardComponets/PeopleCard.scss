.peopleCard {
    width: 320px;
    text-align: center;
    margin-bottom: 20px;
    margin-right: 20px;
    border: 4px solid #3F51B5;
    border-radius: 25px;
    padding-bottom: 10px;
    background: #98dcff;
    box-shadow: 4px 4px 4px #968888;
}

.peopleName {
    background: #9C27B0;
    height: 40px;
    display: flex;
    justify-content: center;
    align-items: center;
    margin-bottom: 20px;
}