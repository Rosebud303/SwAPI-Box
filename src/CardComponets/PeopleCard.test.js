import React from 'react';
import PeopleCard from './PeopleCard';
import { shallow } from 'enzyme';

describe('PeopleCard', () => {
    let wrapper;
    let mockPeople = {name: 'Luke Skywalker', homeworld: 'Tatooine'}

    beforeEach(() => {
        wrapper = shallow(
            <PeopleCard people={mockPeople}
                        key={mockPeople.name}
            />
        )
    })

    it('should match the snap shot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})