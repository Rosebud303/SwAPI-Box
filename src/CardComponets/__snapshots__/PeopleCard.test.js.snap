// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PeopleCard should match the snap shot 1`] = `undefined`;
